config/locales/activerecord.no.yml

```diff
@@ -1,12 +1 @@
-no:
-  activerecord:
-    errors:
-      models:
-        account:
-          attributes:
-            username:
-              invalid: bare bokstaver, tall og understreker
-        status:
-          attributes:
-            reblog:
-              taken: av status eksisterer allerede
```

Comment:
This keys is target of [ignore_unused](https://github.com/tootsuite/mastodon/blob/ef226a6f223ad3bdf662efb0ed75694b3f0bfb56/config/i18n-tasks.yml#L46).